docs/components/board/customlinux.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Configure a customlinux board"
+linkTitle: "customlinux"
+weight: 75
+type: "docs"
+description: "Configure a customlinux board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Susmita
+---
+
+Configure a `customlinux` board to integrate linux boards like the [Pumpkin i500](https://ologicinc.com/portfolio/mediateki500/) into your robot.
+
+Complete the following setup requirements, then move on to configuring your board in the [Viam app](https://app.viam.com):
+
+## Setup requirements
+
+- Create a new robot in the [Viam app](https://app.viam.com) and follow the steps on the [setup tab](/installation/#installation-steps) to [install `viam-server`](/installation/) on your `customlinux` board and connect your board to the Viam app.
+- On your `customlinux` board, create a file in your home folder (<file>/home/root</file>) <file>called board.json</file> with your board's pin mappings:
```

Comment:
```suggestion
- On your `customlinux` board, create a file in your (<file>/home/root</file>) directory called <file>board.json</file> with your board's pin mappings:
```
File started in wrong place, otherwise opt

Reply:
```suggestion
- On your `customlinux` board, create a file in your <file>/home/root</file> directory called <file>board.json</file> with your board's pin mappings:
```